### PR TITLE
Add_stitch_copy

### PR DESCRIPTION
**Pull request by ryantimjohn**
Add a script to copy datasets from stitch

### CODE REVIEW

File: utils/create_views_from_stitch_datasets.ipynb [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fb29b5e9-c78e-4436-826f-58b058a1f095",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "PROJECT_ID = \"[your-project-id]\" \n",
```
Comment (Frydafly) [inline utils/create_views_from_stitch_datasets.ipynb:12]: for my own reference -- what is the project ID for this? (that is in the GCP project?)

File: utils/create_views_from_stitch_datasets.ipynb [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c22d07c8-2bbc-4dc5-9858-d2d254d3d83b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(search_statement)"
```
Comment (Frydafly) [inline utils/create_views_from_stitch_datasets.ipynb:41]: ran this notebook using `bsd-oxfam` and there was no search_statement printed in my notebook -- is that fine, or should there be a flag if that happens?

File: utils/create_views_from_stitch_datasets.ipynb [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    "                        \"\"\"\n",
+    "                    qs.append(current_project_client.query(clone_table_statement))\n",
+    "                    print(f\"Started cloning to bsd-stitch.{dataset_id}.{table.table_id}\\n\")\n",
+    "                num_queries = len(qs)\n",
+    "                num_complete = sum([q.running()==False for q in qs])\n",
+    "                while num_complete < num_queries:\n",
+    "                    print(f\"{num_complete} out of {num_queries} completed!\")\n",
+    "                    sleep(10)\n",
+    "                    num_complete = sum([q.running()==False for q in qs])\n",
+    "                print(f\"\\nFinished {dataset_id}!\")\n",
```
Comment (Frydafly) [inline utils/create_views_from_stitch_datasets.ipynb:94]: @ryantimjohn got this error `DefaultCredentialsError: Could not automatically determine credentials. Please set GOOGLE_APPLICATION_CREDENTIALS or explicitly create credentials and re-run the application. For more information, please see https://cloud.google.com/docs/authentication/getting-started`

Did I miss somewhere I could save the credentials json? @ryantimjohn

**Review by Frydafly — CHANGES_REQUESTED** [review]
hi! I tried running this notebook locally and ran into a few issues. might just be things I don't know, and even so, would be good to add to the documentation so someone like me could run successfully?

**Comment by ryantimjohn** [comment]
@Frydafly this one is meant to be run in a VertexAI notebook where those values are taken care of, will add that to a markdown cell on top and also to README!

Will also need to do some service account work to make these notebooks work. I'll add that to the Markdown file at the top, too!

**Comment by ryantimjohn** [comment]
Screen Shot 2022-06-16 at 8 05 33 PM
Fryda, added a bunch of documentation. Let me know if this works for you!

**Comment by ryantimjohn** [comment]
Screen Shot 2022-06-16 at 8 05 33 PM
@Frydafly, added a bunch of documentation. Let me know if this works for you!

**Comment by Frydafly** [comment]
So perfect!! Thank you

All the best,
Fryda
(she/her)

**Comment by ryantimjohn** [comment]
Also added this documentation to the clone tables script.

**Comment by ryantimjohn** [comment]
And added an explainer to README.md

**Comment by ryantimjohn** [comment]
I'm going to say that this is good now. @Frydafly if you want to try to run this in a VertexAI notebook, please give it a shot!

**Pull request merged**